src/components/Setup/Views/InstallPWA.tsx

```
@@ -194,94 +172,76 @@
+                                        className="mt-4 w-full"
+                                        shadowSize="4"
+                                        variant="purple"
+                                    >
+                                        Got it!
+                                    </Button>
+                                </div>
+                            </Modal>
+                        )}
+                    </>
+                )
+            }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)